src/poliastro/core/perturbations.py

```diff
@@ -254,7 +280,32 @@ def radiation_pressure(t0, state, k, R, C_R, A_over_m, Wdivc_s, star):
     Wdivc_s : float
         total star emitted power divided by the speed of light (W * s / km)
     star: a callable object returning the position of star in attractor frame
-        star position
+        .. py:function:: star() -> position_of_star
+        .. py:class:: Star() -> position_of_star
+
+    Example
+    -------
+    For clarifying the nature of **star** argument in **radiation_pressure()** function, we can look into the following example:
+
+    >>> import functools
+    >>> def normalize_to_Curtis(t0, sun_r):
+    ...     r = sun_r(t0)
+    ...     return 149600000 * r / norm(r)
+    >>> def sun_r():
+    ...     j_date = 2_438_400.5 * u.day
+    ...     tof = 600 * u.day
+    ...     return build_ephem_interpolant(Sun, 365 * u.day, (j_date, j_date + tof), rtol=1e-2)
+    >>> sun_normalized = functools.partial(normalize_to_Curtis, sun_r=sun_r)
+    >>> star=sun_normalized     # passing created callable function to star
```

Comment:
This line is not needed? It's just renaming a variable

Reply:
Yes sure but this was important as the argument passed in the requested function is `star` so that it makes sense to the user reading the example :)